Utilities/Config.h

```diff
@@ -309,6 +312,73 @@ namespace cfg
 		}
 	};
 
+	// Float entry with custom Min/Max range.
+	template <s64 Min, s64 Max>
```

Comment:
floats in templates should be supported now.

Reply:
I have tried, it fails compilattion on GCC/clang.

Reply:
I'll check, maybe newer compilers work